src/app/furniture.ts

```diff
@@ -0,0 +1,5 @@
+export interface Furniture {
+  furnitureId: number;
```

Comment:
if the id is of the entity, just name it as id.

Reply:
Changed